langchain/src/vectorstores/vercel_postgres.ts

```diff
@@ -247,19 +247,42 @@ export class VercelPostgres extends VectorStore {
     filter?: this["FilterType"]
   ): Promise<[Document, number][]> {
     const embeddingString = `[${query.join(",")}]`;
-    const _filter = filter ?? "{}";
+    const _filter = filter ? filter : {};
+    const whereClauses = [];
+    const values = [embeddingString, k];
+    let paramCount = 2;
+
+    for (const [key, value] of Object.entries(_filter)) {
+      if (value.hasOwnProperty("in")) {
```

Comment:
So this would look like 
```
{
  "key": {
    "in": ["item", "item2"]
}
```
right?

Can you add an integration test?

Reply:
Yep - that's right! And happy to add tests and docs, will add this later today!

Reply:
Sure, will do now!